<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QMainWindow" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendNewMsgBtn">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>740</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newMsgLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>730</y>
      <width>560</width>
      <height>50</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type your message here...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msgsBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>720</width>
      <height>700</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
